src/openforms/conf/base.py

```diff
@@ -1182,6 +1182,10 @@
     "LOG_OUTGOING_REQUESTS_MAX_AGE", default=7 * 24
 )  # number of hours
 
+# Pdok api locatie server urldeddddddd77487
+PDOK_LOCATIE_SERVER_URL = config(
+    "PDOK_LOCATIE_SERVER_URL", "https://api.pdok.nl/bzk/locatieserver/search/v3_1/"
+)
```

Comment:
"urldeddddddd77487" ?

Locatieserver is one word in Dutch. Please correct that in all locations. I like that you stuck with the name of the service.

Reply:
I have absolutely no clue how this happend and how I missed it.